src/features/click-to-play.js

```diff
@@ -1229,6 +1623,53 @@ function getLearnMoreLink (mode) {
     return linkElement
 }
 
+/**
+ * Reads and stores a set of styles from the original tracking element, and then returns it.
+ * @param {Element} originalElement Original tracking element (ie iframe)
+ * @param {DuckWidget} widget The widget Object.
+ * @returns {{[key: string]: string[]}} Object with styles read from original element.
+ */
+function getOriginalElementStyle (originalElement, widget) {
+    if (widget.originalElementStyle) {
+        return widget.originalElementStyle
+    }
+
+    const stylesToCopy = ['display', 'visibility', 'position', 'top', 'bottom', 'left', 'right',
+        'transform', 'margin', 'width', 'max-width', 'height', 'max-height']
+    widget.originalElementStyle = {}
+    const allOriginalElementStyles = getComputedStyle(originalElement)
+    for (const key of stylesToCopy) {
+        widget.originalElementStyle[key] = [
+            allOriginalElementStyles[key],
+            originalElement.style.getPropertyPriority(key)
+        ]
+    }
+
+    // Try to get original height/width styles from element,
+    // otherwise measures current size and stores that
+    const { height: heightViewValue, width: widthViewValue } = originalElement.getBoundingClientRect()
+    let styleHeight = originalElement.height
+    styleHeight += styleHeight.search(/^\d+(\.\d+)*$/g) > -1 ? 'px' : ''
+    widget.originalElementStyle.height[0] = styleHeight || widget.originalElementStyle.height[0] || heightViewValue
```

Comment:
The idea here was to try and make the placeholder reproduce the original sizing of the embedded video, and resize if the original would resize, as it was discussed in the Design Review. But I'm starting to see this being increasingly difficult to maintain and not always getting the results we would like.
I'll fix the size again and ping design about it. They were already aware it was going to be hard to make it work properly.

Just one added note regarding the `If the regex doesn't match we're going to use a raw unit value for the height is that intended?` question. I was checking both the `height` property and the `height` style, because they can have different values, or could be missing, and then `getBoundingClientRect()` would be a fallback. And the height property will return a value without pixels sometimes, that's why I was using the regex to check if only numbers and `.` existed, otherwise we would add the `px`. For other values like `%`, nothing changed. And finally, `getComputedStyle()` would always return the value with its unit, so adding `px` was not necessary. Yes, this is as confusing as it sounds, and that's why I agree with going back to a fixed size, and we can iterate based on user feedback later.
![image](https://user-images.githubusercontent.com/14364481/205074321-688afbff-7945-4144-8c41-08368d59b78e.png)

![image](https://user-images.githubusercontent.com/14364481/205074991-7f6e85e2-df9b-4001-acb8-ef7b38a801c3.png)